httpclient5/src/main/java/org/apache/hc/client5/http/entity/mime/ByteArrayBody.java

```diff
@@ -56,11 +56,24 @@
      */
     public ByteArrayBody(final byte[] data, final ContentType contentType, final String filename) {
         super(contentType);
-        Args.notNull(data, "byte[]");
-        this.data = data;
+        this.data  =Args.notNull(data, "byte[]");
         this.filename = filename;
     }
 
+    /**
+     * Public constructor that creates a new ByteArrayBody.
+     *
+     * @param contentType  the {@link ContentType}
+     * @param data the array of byte.
+     *
+     * @since 5.2
+     */
+    public ByteArrayBody(final byte[] data, final ContentType contentType) {
+        super(contentType);
+        this.data = Args.notNull(data, "byte[]");
+        this.filename = null;
+    }
```

Comment:
Why the code duplication instead of passing to the previous ctor?

Reply:
HI @michael-o.  True. Good catch. 
Changed.
TY